boards/radxa/default.nix

```diff
@@ -0,0 +1,16 @@
+{ rockchipRK399 }:
+
+{
+  radxa-RockPi4C = rockchipRK399 {
+    boardIdentifier = "radxa-RockPi4C";
+    defconfig = "rock-pi-4c-rk3399_defconfig";
+    SPISize = 4 * 1024 * 1024; # 4 MiB
```

Comment:
> A 4MB(32Mbit) SPI flash is pasted by default, for future SPI booting support

Aww... This puts a damper on some plans I had that required *just a bit more space*... But good to know this before going forward.

(You don't need to do anything about this...)

Reply:
They're easy to replace, they have a big footprint (and some were even delivered without a flash).